### ISSUE TITLE
@JiahuiYu Regarding Discriminator conv for deefillv2.

### ISSUE THREAD

**Issue by khemrajrathore** [JiahuiYu/generative_inpainting#165]
def dis_conv(x, cnum, ksize=5, stride=2, name='conv', training=True):
    x_shape=x.get_shape().as_list()
    print("x_shape................................",x_shape)
    #print("The shape for x_shape ",x_shape)
    w=tf.get_variable(name=name+'_w',shape=[ksize,ksize,x_shape[-1]]+[cnum])
    w=conv2d_spectral_norm(w,cnum,ksize,name=name)   
    x = tf.layers.conv2d(x,w,strides=[1,stride,stride,1],padding='SAME',data_format='channels_last')
    bias=tf.get_variable(name=name+'_bias',shape=[cnum])
    return  tf.nn.leaky_relu(x+bias)

This is my code for dis_conv. I have used spectral norm from neuralgyms dev branch. The w tensor returned has different shape than expected.

**Comment by JiahuiYu** [comment]
Maybe the shape of weight should be k, k, output channel, input channel? I am not sure please check the tensorflow API carefully.
